oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollector.java

```diff
@@ -31,6 +32,34 @@ public interface FullGCStatsCollector {
      */
     void documentRead();
 
+    /**
+     * Total No. of properties detected as garbage during a given GC phase
+     * @param mode GC phase
+     * @param numProps no. of garbage properties found in current cycle
+     */
+    void candidateProperties(GCPhase mode, long numProps);
+
+    /**
+     * Total No. of documents detected as garbage during a given GC phase
+     * @param mode GC phase
+     * @param numCommits no. of garbage documents found in current cycle
+     */
+    void candidateDocuments(GCPhase mode, long numCommits);
```

Comment:
What is the use case of this counter? I see it is currently only used in `collectUnmergedBranchCommits` hence seems a bit asymmetric. There already is a counter for how many documents are read (`documentRead`), is there an advantage of having `candidateDocuments` too?

Reply:
This is wrong, yes. `collectUnmergedBranchCommits` deletes revisions, so I will change this to the one that counts deleted revisions. Probably this one is unneeded then, as there is already one that counts deletedDocuments (orphan).